src/test/java/seedu/recipe/commons/util/AppUtilTest.java

```diff
@@ -9,7 +9,7 @@ public class AppUtilTest {
 
     @Test
     public void getImage_exitingImage() {
-        assertNotNull(AppUtil.getImage("/images/address_book_32.png"));
+        assertNotNull(AppUtil.getImage("/images/cookHub.png"));
```

Comment:
Yay! Looks good! Can just check if it passes checkstyle first so later we don't have to do it one shot